cms/static/js/views/active_video_upload_list.js

```diff
@@ -198,6 +212,11 @@ define([
                 this.setStatus(data.cid, ActiveVideoUpload.STATUS_FAILED);
             },
 
+            getMaxFileSizeInBytes: function() {
+                var CONVERSION_FACTOR_GBS_TO_BYTES = 1000 * 1000 * 1000;
```

Comment:
We are initialising here because of readability, we do need to know what is `1000 * 1000 * 1000`